app/app.py

```diff
@@ -10,14 +10,25 @@
 
 load_dotenv()
 
+
+
 get_bearer_token = HTTPBearer(auto_error=False)
 known_tokens = set([os.getenv('BearerToken')])
 
 def transcribe_audio(file_path: str) -> str:
-    model = whisper.load_model("large")
+    model = whisper.load_model("tiny")
     text = model.transcribe(file_path, fp16=False)
-    os.remove(file_path)
     return text
+    
+# Use Faster-whisper for much better speed compared to whisper tiny
+def transcribe_audio_faster(file_path: str) -> str:
+    from faster_whisper import WhisperModel
+    model_size = "tiny"
+    #run on CPU with INT8
+    model = WhisperModel(model_size, device="cpu", compute_type="int8")
+    segments, info = model.transcribe(file_path, beam_size=5)
+    for segment in segments:
```

Comment:
Use a generator instead of a loop. Don't store the data during operation in memory